src/app/components/daily-review/daily-review.component.html

```diff
@@ -1,30 +1,79 @@
-  <div class="container">
-    <h2>Daily Review {{this.entryDate}}</h2>
-    <div class="flex-parent-container">
-
-      <app-wellness-rating-form
-      [wellnessRatingForm]="wellnessRatingForm"
-      [previousDailyReview]="previousDailyReview">
-      </app-wellness-rating-form>
-     
-
-      <app-habit-review-form 
-        [habitReviewForm]="habitReviewForm"
-        [noHabits]="noHabits"
-        [habits]="habits">
-      </app-habit-review-form>
-
-      <app-guided-journal-form
-      [guidedJournalForm]="guidedJournalForm"
-      [previousDailyReview]="previousDailyReview"
-      [journalEntries]="journalEntries"
-      >
-      </app-guided-journal-form>
-  
-  
-      
-    </div>
-    <button class="btn btn-primary" (click)="submit()" [hidden]="previousDailyReview">Submit</button>
-    
-    
-  </div>
\ No newline at end of file
+<div class="container">
+  <h2>Daily Review {{this.entryDate}}</h2>
+  <ng-template #modalTemplate let-modal>
+    <form *ngIf="modalForm" [formGroup]="modalForm" (ngSubmit)="update()">
+      <div class="modal-header">
+        <h4 class="modal-title">Actions</h4>
+        <button type="button" class="btn-close" aria-label="Close" (click)="modal.dismiss('Cross click')"></button>
+      </div>
+      <div class="modal-body">
+        <div class="container-lg">
+          <div class="table-responsive">
+            <div class="table-wrapper">
+              <div class="table-title">
+                <div class="row">
+                  <div class="col-sm-8">
+                    <h2></h2>
+                  </div>
+                  <div class="col-sm-4">
+                  </div>
+                </div>
+              </div>
+              <table class="table table-bordered">
```

Comment:
Modern pages don't use tables for layout. I'm hoping the CSS course work I shared with you will elaborate on this more.